docs/src/modules/utils/helpers.js

```diff
@@ -54,6 +97,11 @@ export function getDependencies(raw, reactVersion = 'latest') {
       deps[name] = versions[name] ? versions[name] : 'latest';
     }
   }
+
+  if (codeLanguage === 'TS') {
```

Comment:
I think that it would be great to use `CODE_VARIANTS.TS` so we can refactor the code more easily.